internal/app/mapper/mapper_test.go

```diff
@@ -34,7 +35,25 @@ var postivetests = []TableEntry{
 			getAnyMatch(true),
 			getResultStringFragment(),
 			clusterTypeURL,
-			stringfragment,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestTypeMatch Matches with a single typeurl",
```

Comment:
Do we need to add a test case for ordering of the type url? (second match vs first)

Reply:
good idea. Done